js/krakenfutures.js

```diff
@@ -434,6 +435,12 @@ module.exports = class krakenfutures extends Exchange {
         return this.parseOrderBook (response['orderBook'], symbol, timestamp);
     }
 
+    async fetchTicker (symbol, params = {}) {
```

Comment:
Hello @youknowone, thanks for your contribution, however, we should apply this fix in the base method (inside Exchange.js). Don't worry I will fix it